src/docfx/docset/Document.cs

```diff
@@ -20,6 +20,16 @@ internal class Document
         /// </summary>
         public ContentType ContentType { get; }
 
+        /// <summary>
+        /// Gets the MIME type specifed in YAML header or JSON $schema.
+        /// </summary>
+        public string Mime { get; }
+
+        /// <summary>
+        /// Gets the schema identified by <see cref="Mime"/>.
+        /// </summary>
+        public DataSchema Schema { get; }
```

Comment:
>DataSchema [](start = 15, length = 10)

it's confusing me for data schema and data schema attribute....

Reply:
rename to Schema?

---
In reply to: [212891103](https://github.com/dotnet/docfx/pull/3291#discussion_r212891103) [](ancestors = 212891103)